{{cookiecutter.project_dirname}}/Dockerfile

```diff
@@ -1,16 +1,18 @@
-ARG PYTHON_VERSION=3.10
-FROM python:${PYTHON_VERSION}-slim-bullseye AS base
+FROM python:3.11-slim-bullseye AS base
```

Comment:
Let's keep track of the reasons that led to the removal of the parametric Python version. This feature may be reintroduced at a later stage, after having deepened all the possible issues.